addons/l10n_sa_edi/models/account_tax.py

```diff
@@ -0,0 +1,78 @@
+from odoo import fields, models
+
+
+class AccountTax(models.Model):
+    _inherit = 'account.tax'
+
+    l10n_sa_is_retention = fields.Boolean("Is Retention", default=False,
+                                          help="Determines whether or not a tax counts as a Withholding Tax")
+
+    l10n_sa_exemption_reason_code = fields.Selection(string="Exemption Reason Code",
+                                                     selection=[
+                                                         ('VATEX-SA-29',
```

Comment:
We could put the selection in a global var, so we don't need to repeat it entirely.

Reply:
Done